apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/eobByPatientIdAll.json

```diff
@@ -5136,7 +5136,7 @@
             } ]
           },
           "amount" : {
-            "value" : 75.0,
+            "value" : 47.84,
```

Comment:
Assuming these were changed to be unique values; it's a good idea. We should ideally have all unique values for each input to ensure we don't have the wrong thing going in the wrong place (if not just verifying the input path value)